### PR TITLE
Remove "preferGlobal" from "package.json" file

### PR DESCRIPTION
**Pull request by mdentremont**
- Add instructions for installing the cli tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,12 @@
 npm install marked --save
 ```
 
+or if you want to use the `marked` CLI tool:
```
Comment (jasonkarns) [inline README.md]: maybe worth noting that global install isn't necessary when using npm run-scripts?

Comment (mdentremont) [inline README.md]: Done

**Comment by mdentremont** [comment]
@chjj @matt- @parleur - This very small pull-request will help rid ember.js of some deprecation warnings, any attention would be much appreciated 👍

**Comment by mdentremont** [comment]
@chjj @matt- @parleur - This very small pull request will help rid Ember.js of some deprecation warnings, any attention would be much appreciated 👍

**Comment by mdentremont** [comment]
@chjj @matt- @parleur any chance this could be looked at?

**Comment by mdentremont** [comment]
@chjj @matt- @parleur any chance this could be looked at?

**Comment by mdentremont** [comment]
@chjj @matt- @parleur This has been open and ready to go for months, could you please take a look and merge?

**Comment by mdentremont** [comment]
@matt- Thanks! ..you know you wanna hit that green button now, don't ya!

**Comment by mdentremont** [comment]
@matt- Thanks! ..you know you wanna hit that green button now, don't ya!

**Comment by joshbruce** [comment]
Fixes #870

**Pull request merged**